docs/diagrams/ModelClassDiagram.puml

```diff
@@ -43,11 +47,14 @@ UniquePersonList --> "~* all" Person
 Person *--> Name
 Person *--> Phone
 Person *--> Email
-Person *--> "*" Event
 Person *--> "*" Role
 
 UniqueEventList --> "~* all" Event
-Event *--> "1" EventName
```

Comment:
To my knowledge the EventName still exists, it's just generated upon construction instead of parsed, so it should still be there, with dependency on Sport and Faculty maybe?

Perhaps we could  add a layer of inheritance for all the role types we have as well